qemu/tests/balloon_check.py

```diff
@@ -58,7 +58,7 @@ def get_ballooned_memory(self):
         return ballooned_mem
 
     @error_context.context_aware
-    def memory_check(self, step, ballooned_mem):
+    def memory_check(self, step, ballooned_mem, balloon_threshold=100):
         """
         Check memory status according expect values
 
```

Comment:
Please add explanation about new parameter "balloon_threshold" in docstring.

Reply:
Modified, thanks